tests/core_tests.py

```diff
@@ -804,15 +804,15 @@ def test_import_csv(self):
         test_file.write('paul,2\n')
         test_file.close()
         main_db_uri = db.session.query(
-            models.Database.sqlalchemy_uri)\
+            models.Database)\
```

Comment:
If you move the closing parens on the line below you can avoid the backslash

Reply:
Done